internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -384,6 +428,11 @@ func (h *Host) CreateContainer(ctx context.Context, id string, settings *prot.VM
 		}()
 	}
 
+	user, groups, umask, err := getUserInfo(settings.OCISpecification)
```

Comment:
not a fan that we'll call this every time, regardless of whether we need it or not... maybe it's worth moving the logic to the enforcement point itself and if we need to export a few methods, that's fine.

Reply:
As discussed, this has been mitigated by moving it to the enforcer interface.